back-end/API/app/controllers/enterprise_controller.py

```
@@ -38,33 +38,34 @@ def getEnterpriseByEmployer(id):
-        enterprise = enterprise_service.getEnterprise(employer.enterpriseId)
-
         data = request.get_json()
 
-        if current_user.isModerator and data["id"] != None:
+        if current_user.isModerator and data.get("id") is not None:
             enterprise = enterprise_service.getEnterprise(data["id"])
             if enterprise:
                 enterprise_service.updateEnterprise(data)
                 return jsonify({'message': 'enterprise updated'})
+            else:
+                logger.warn(f'Enterprise not found with id : {data["id"]}')
```

Ruff (0.15.2)

[warning] Logging statement uses `warn` instead of `warning`

Convert to `warning`

(G010)